codecov.yml

```diff
@@ -1 +1,15 @@
 comment: false
+
+coverage:
+  status:
+    project:  # settings affecting project coverage
+      default:
+        target: auto  # auto % coverage target
+        threshold: 1%  # allow for 1% reduction of coverage without failing
```

Comment:
What's the behavior of this?

I'm ok with bot leaving comments if coverage goes down but I don't think we should auto-block PRs based on this.

Reply:
I was just aligning our codecov conf with BuildKit one. I think for BuildKit we added this years ago because reports were inaccurate. We could try removing it.

Reply:
I'm just not sure what it even does. Not just the `1%` but this block in general.